src/main/kotlin/pt/up/fe/ni/website/backend/model/Project.kt

```diff
@@ -0,0 +1,16 @@
+package pt.up.fe.ni.website.backend.model
+
+import org.springframework.data.jpa.domain.support.AuditingEntityListener
+import javax.persistence.Entity
+import javax.persistence.EntityListeners
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+
+@Entity
+@EntityListeners(AuditingEntityListener::class)
```

Comment:
What is this for?

Reply:
This was used for `publishDate` and `lastUpdatedAt` in the Post. Do we need it here?

Reply:
No xD

Reply:
Thanks!